benchmark/pkg/runselector/schemes/last_n_runs_test.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schemes
+
+import (
+	"testing"
+)
+
+const (
+	lastNhoursJob = "ci-kubernetes-e2e-gce-scalability"
+	numRuns       = 3
+)
+
+func TestGetLastNJobRuns(t *testing.T) {
```

Comment:
This still depends on something on the network. I need to be able to run this test from my laptop when sitting in the airplane.